src/tools.json

```diff
@@ -3,27 +3,27 @@
         "description": "odo CLI",
         "vendor": "Red Hat, Inc.",
         "name": "odo",
-        "version": "2.5.0",
-        "versionRange": "^2.5.0",
-        "versionRangeLabel": "version >= 2.5.0",
+        "version": "2.5.1",
+        "versionRange": "^2.5.1",
+        "versionRangeLabel": "version >= 2.5.1",
         "dlFileName": "odo",
         "filePrefix": "",
         "platform": {
             "win32": {
-                "url": "https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v2.5.0/odo-windows-amd64.exe.zip",
-                "sha256sum": "a7856bb39a279ce3484bad13e383d5788d1e6e105d3eec028dbb7843e1c2284f",
+                "url": "https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v2.5.1/odo-windows-amd64.exe.zip",
+                "sha256sum": "98d11fed4eb67d8182d1c83f0ca5208d8e75e141387b1794bdec92946e4e318c",
                 "dlFileName": "odo-windows-amd64.exe.zip",
-                "cmdFileName": "odo.exe"
+                "cmdFileName": "odo-windows-amd64.exe"
```

Comment:
@msivasubramaniaan do we need this to be changed too ? cc @dgolovin

Reply:
Should stay as it was. Fixed.

Reply:
ups, I checked old URL not new one.